### DISCUSSION TITLE
Is it possible to perform a perspective transform on an animated sprite?

### DISCUSSION THREAD

**Discussion by batman4444** [pixijs/pixijs#8845] [Q&A] [answered]
I am trying to accomplish a sprite flipping animation with a bit of 3d perspective on it. 

I found several posts indicating that pixi does not support perspective transformations, however they were all from several years ago.  I am wondering if it is now currently supported.

**Comment by bigtimebuddy** [comment, marked as answer]
There's a plugin that can handle this, but it's still not a feature available by default. See https://github.com/pixijs/pixi-projection
